ui/apps/platform/src/hooks/useAnalytics.ts

```diff
@@ -87,6 +88,11 @@ export const searchCategoriesWithFilter = [
     'CLUSTER CVE FIXABLE',
     'CVSS',
     'Node Top CVSS',
+    'Category',
+    'Severity',
+    'Lifecycle Stage',
+    'Resource Type',
+    'Inactive Deployment',
```

Comment:
The "duplicate" of 'Severity' and 'SEVERITY' here makes me wish that we enforced a `toUpper()` on the values sent to Amplitude when telemetry was first implemented.

Reply:
Yeah, I caught that as well. Taking it a step further, it might be beneficial to limit the variations as well. We can look into this one a bit more post-4.6

Reply:
For what it is worth, unit tests check naming conventions (with exceptions) for policy criteria:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Policies/Wizard/Step3/policyCriteriaDescriptors.test.ts